style.css

```diff
@@ -765,3 +765,137 @@ button {
   position: absolute;
   left: 133px;
 }
+
+/* contact me form */
+
+.contact-me {
+  display: flex;
+  padding-top: 315px;
+  justify-content: center;
+}
+
+.form-items-list {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-me-para {
+  width: 321px;
+  height: 216px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  color: #172b4d;
+  padding-top: 217px;
+  padding-left: 18px;
+  font-size: 32px;
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  line-height: 44px;
+  text-align: center;
+}
+
+input,
+textarea {
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+}
+
+input {
+  color: #979493;
+  width: 287px;
+  position: relative;
+  left: -25px;
+  top: 47px;
+  padding: 26px;
+  margin-bottom: 5px;
+}
+
+.name input:invalid {
+  border-color: rgb(253, 6, 6);
+}
+
+.text-area {
+  color: #3c3a39;
+  background-color: #fbf8f7;
+  width: 314px;
+  padding-left: 21px;
+  margin-top: 77px;
+  margin-bottom: 77px;
+  outline: none;
+  position: absolute;
+  left: 19px;
+  right: 12px;
+  top: 2570px;
+  bottom: -794px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+}
```

Comment:
- [ ] Great job so far. Your form is working and looks great however please I would suggest that you decrease the margin-top of the textarea field as it looks too far away compared to that in the figma design template. As the requirements state that we have to stick to the figma design as closely as possible.

Figma design:

<img width="472" alt="Screenshot 2023-04-07 at 9 59 17 PM" src="https://user-images.githubusercontent.com/44798044/230649346-2e096ed1-cc7b-4d59-8da1-a9a87503a115.png">

Your design:

<img width="347" alt="Screenshot 2023-04-07 at 9 59 34 PM" src="https://user-images.githubusercontent.com/44798044/230649480-033a2da0-c4ea-46dd-8d83-6717eb92b343.png">